src/ploneorg/addonlisting/contents.py

```diff
@@ -97,8 +97,8 @@ class FilterForm(form.SchemaForm):
     schema = IFilterForm
     fields = field.Fields(IFilterForm)
```

Comment:
you try to execute code on an class attribute scope, you are not in a nested method.

It is a mater of the scope: This is an class attribute declaration not a class instance attribute or even a method scope attribute assignment. That could not work at all.